prov/verbs/src/verbs_domain.c

```diff
@@ -423,9 +424,17 @@ fi_ibv_domain(struct fid_fabric *fabric, struct fi_info *info,
 
 	*domain = &_domain->domain_fid;
 	return 0;
-err3:
+err5:
 	if (_domain->rdm)
 		rdma_destroy_event_channel(_domain->rdm_cm->ec);
+err4:
+	if (ibv_dealloc_pd(_domain->pd))
+		VERBS_INFO_ERRNO(FI_LOG_DOMAIN, "ibv_dealloc_pd", errno);
+err3:
+	if (_domain->rdm) {
+		_domain->rdm_cm->fi_ibv_rdm_tagged_cm_progress_running = 1;
```

Comment:
Shouldn't this be set to 0?

Reply:
Sure, it should be set to 0.

The original changes that fix the reported problem in the PR #3268. Wrong copy-pasting :(
Thank you for being attentive.